### DISCUSSION TITLE
Fixed RNA Profiling data analysis

### DISCUSSION THREAD

**Discussion by yguillen** [satijalab/seurat#6494] [Q&A]
Hi all,
We are analyzing single cell data obtained through the new FRP protocol from 10X (https://www.10xgenomics.com/products/fixed-rna-profiling). Unlike conventional 3' method, this is a probe-based method, meaning that only a collection of genes are surveyed. After analysing this data following typical downstream steps (QC, PCA, UMAP integration, etc) we suspect that there is a bias driven by the restricted collection of probes that are surveyed. Should this type of data be analysed with a probe-based-adapted seurat pipeline? Have you ever implemented seurat in this kind of data?
Thanks!

**Comment by liliay** [comment]
Hi @yguillen, have the same question about FRP data analysis. Did you manage to have any clue on this ? 

Thanks
